clang/runtime/dpct-rt/include/dpl_extras/memory.h.inc

```diff
@@ -50,13 +50,31 @@
 // DPCT_LABEL_END
 namespace dpct {
 namespace detail {
-// DPCT_LABEL_BEGIN|__buffer_allocator|dpct::detail
+template <typename T>
+// DPCT_LABEL_BEGIN|make_allocatable|dpct::detail
 // DPCT_DEPENDENCY_EMPTY
```

Comment:
The code depends on the dpct::byte_t, please help update it.

Reply:
Dependence information has been updated.